services/volume-inspector/main.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
It's ok to put the whole code in a single file. In this case a separate api.go file doesn't make much sense.